projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/launchpad-save-modal.php

```diff
@@ -22,8 +22,9 @@ function add_launchpad_options() {
 	}
 
 	$launchpad_options = array(
-		'launchpadScreenOption' => get_option( 'launchpad_screen' ),
-		'siteIntentOption'      => get_option( 'site_intent' ),
+		'launchpadScreenOption'       => get_option( 'launchpad_screen' ),
+		'siteIntentOption'            => get_option( 'site_intent' ),
+		'hasNeverPublishedPostOption' => get_option( 'has_never_published_post' ),
```

Comment:
That's unrelated to this PR, but I'm discovering this `has_never_published_post` option today. It would be really nice to use prefixed option names like `wpcom_has_never_published_post` or `jetpack_has_never_published_post` for such things, to avoid any potential conflicts with other plugins.

Edit: I ended up posting about it here:
p1676654103372569-slack-C03N25JPCE4